app/controllers/content_types/modular_controller.rb

```diff
@@ -87,6 +87,120 @@ def one_column
     end
     # rubocop:enable Metrics/MethodLength
 
+    # GET /modular/two_column
+    # Maps to Content Types/Modular/2 Column in Cascade.
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+    def two_column
+      # Cascade Data Models
+      @configuration_set = ConfigurationSet.two_column
+      @metadata_set = MetadataSet.page(title: 'Modular Two Column')
+      @data_definition = DataDefinitions::TwoColumn.default
+
+      # Set dynamic values.
+      if params[:masthead] == 'branded-old'
+        @data_definition.set_value(:masthead_type, 'Branded Masthead')
+      elsif params[:masthead] == 'slider-old'
+        @data_definition.set_value(:masthead_type, 'Slider')
+      elsif params[:masthead] == 'slider'
+        @data_definition.set_value(:masthead_type, 'Slider - New')
+      else
+        @data_definition.set_value(:masthead_type, 'Branded - New')
+      end
```

Comment:
I agree in future something may need to be done. For now, I think this is manageable and easier to comprehend.

The naming mismatch is brought over from Cascade. We need to map URL param values (with their URL-encoded format) to the Cascade labels we used.

I'll add a comment clarifying this.